### PR TITLE
Squash ts errors

### PR DESCRIPTION
**Pull request by TomPretty**
## What are you doing in this PR?

Squash ts errors

### CODE REVIEW

File: support-frontend/assets/helpers/urls/externalLinks.ts [hunk 1]
```diff
@@ -73,7 +71,6 @@ function buildParamString(
 }
 
 function getLegacyPaperAndDigitalLink(
```
Comment (GHaberis) [inline support-frontend/assets/helpers/urls/externalLinks.ts:73]: I'm wondering whether this `getLegacyPaperAndDigitalLink` function is even used anymore? It used to generate a link to point the depreacted subscribe site.

Comment (TomPretty) [reply support-frontend/assets/helpers/urls/externalLinks.ts:73]: Yeah I'll go an clean up the unused functions in this file

Comment (TomPretty) [reply support-frontend/assets/helpers/urls/externalLinks.ts]: done!

**Review by GHaberis — APPROVED** [review]
looks good 👏

**Pull request merged**